src/model/user.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn({
+    type: 'bigint',
+    name: 'user_id',
+  })
+  id: number;
+
+  @Column({
+    name: 'email_address',
+    nullable: false,
+    default: '',
+  })
+  email: string;
+
+  @Column({
+    name: 'password',
+    nullable: false,
+    default: '',
+  })
+  password: string;
+
+  @Column({
+    name: 'name',
+    nullable: false,
+    default: '',
+  })
+  name: string;
+
+  @Column({
+    name: 'surname',
+    nullable: false,
+    default: '',
+  })
+  surname: string;
+
+  @Column({
+    name: 'date_of_birth',
+    nullable: false,
+    default: '',
+  })
+  dateOfBirth: string;
+
+  @Column({
+    name: 'address',
+    nullable: false,
+    default: '',
+  })
+  address: string;
```

Comment:
I suggest to read about database normalization -> https://www.sqlpedia.pl/projektowanie-i-normalizacja-bazy-danych/
This will help you solve problem with the address column. Additionally some tips you can find here -> 
https://stackoverflow.com/questions/15863981/normalizing-member-address